lib/ExpensiMark.js

```diff
@@ -196,7 +196,7 @@ export default class ExpensiMark {
                 pre: inputString => inputString.replace('<br></br>', '<br/>').replace('<br><br/>', '<br/>'),
 
                 // Include the immediately followed newline as `<br>\n` should be equal to one \n.
-                regex: /<br(?:"[^"]*"|'[^']*'|[^'"><])*>(?![^<]*(<\/pre>|<\/code>))\n?/gi,
```

Comment:
I had to delete this part to make sure new lines are properly replaced within `<pre>` tag. Otherwise, we're loosing the last new line in the code block.

Here's an example text where it wouldn't work correctly before this change: 
```
<pre>a<br />a<br />a</pre>
```

We're getting the following output string: 
``````
```
a
aa
```
``````

<img width="707" alt="Screenshot 2021-08-01 at 18 31 41" src="https://user-images.githubusercontent.com/3853003/127780138-296950e5-e70a-4285-a2c4-407e6d619b73.png">